packages/compartment-wrapper/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@agoric/compartment-wrapper",
+  "version": "0.0.0+1-dev",
+  "description": "impose inescapable options on a Compartment",
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "type": "module",
+  "main": "./dist/compartment-wrapper.cjs",
+  "module": "./compartment-wrapper.js",
+  "browser": "./dist/compartment-wrapper.umd.js",
+  "exports": {
+    "import": "./compartment-wrapper.js",
+    "require": "./dist/compartment-wrapper.cjs",
+    "browser": "./dist/compartment-wrapper.umd.js"
+  },
+  "scripts": {
+    "build": "rollup --config rollup.config.js",
+    "depcheck": "depcheck",
+    "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint --fix '**/*.js'",
+    "test": "tap --no-esm --no-coverage --reporter spec 'test/**/*.test.js'"
+  },
+  "dependencies": {
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-prettier": "^6.9.0",
+    "eslint-plugin-eslint-comments": "^3.1.2",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-prettier": "^3.1.2",
+    "@rollup/plugin-node-resolve": "^6.1.0",
+    "rollup-plugin-terser": "^5.1.3",
+    "ses": "^0.9.0+1-dev",
```

Comment:
Should be `^0.9.1` I think. Would not hurt to also have a `peerDependency`.

Reply:
Ooh, yes, that is the right answer. I'll add that `peerDependency` to `@agoric/import-bundle` where this code will live.